packages/react/.storybook/_container.scss

```diff
@@ -1,4 +1,4 @@
 $css--font-face: true;
 $css--reset: true;
 
-@import '~carbon-components/src/globals/scss/styles.scss';
```

Comment:
This should already resolve to the local copy already because `~carbon-components` points to the hoisted `carbon-components` which is a symlink 👍